build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         classpath 'com.palantir.gradle.revapi:gradle-revapi:1.7.0'
         classpath 'com.netflix.nebula:gradle-dependency-lock-plugin:7.0.1'
         classpath 'com.palantir.baseline:gradle-baseline-java:5.7.0'
-        classpath 'com.palantir.gradle.gitversion:gradle-git-version:0.15.0'
+        classpath 'com.palantir.gradle.gitversion:gradle-git-version:3.0.0'
```

Comment:
@ash211 do you know why this wasn't excavated?

Reply:
https://github.com/palantir/conjure-java-runtime/pull/2432 got stuck